css-box-model/styles.css

```diff
@@ -0,0 +1,10 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 16px 6px 16px 6px;
```

Comment:
This will apply `16px` top and bottom and `6px` left and right padding. It should be the opposite `6px` top and bottom and `16px` left and right padding.

Reply:
got it makes sense heres an updated photo of the changes
![Screenshot (68)](https://user-images.githubusercontent.com/51901536/129949366-9f0f2b4e-de97-4120-a43a-0a11f3887f42.png)